carveracontroller/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.0'    # This is changed dynamically by poetry-dynamic-versioning
+__version__ = '0.7.2'    # This is changed dynamically by poetry-dynamic-versioning
```

Comment:
Since this is updated by poetry-dynamic-versioning best to keep it at 0.0.0 to keep it obvious that it shouldn't be manually updated

```suggestion
__version__ = '0.0.0'    # This is changed dynamically by poetry-dynamic-versioning
```

Reply:
Oh yeah had to do this since ios is not completly integrated with the build system, reverted for now.